src/lib/components/DashNgl.react.js

```diff
@@ -0,0 +1,253 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Stage, Selection} from 'ngl';
+
+/**
+ * Dash ngl component
```

Comment:
Please include a longer description here - you can take a look at the other components for some inspiration.